.gitignore

```diff
@@ -15,3 +15,4 @@
 __pycache__
 /rsmtool.sublime-workspace
 /rsmtool.sublime-project
+/.condaenv
```

Comment:
Nitpick: Some of these entries might be better-suited in `.git/info/exclude` (that is, custom repository-level ignores) since they are specific to how a particular user names a Sublime project (or even uses Sublime), manages Conda envs, etc. The last here might be a bit confusing to a new developer wondering where this file comes from. Anyway, just something to think about.

Reply:
YesI That is a good call - developer-specific stuff should not go in `.gitignore`. Let me clean this up.